src/main/java/io/specto/hoverfly/junit/core/HoverflyConfig.java

```diff
@@ -32,6 +32,7 @@
     protected String destination;
     protected String proxyCaCert;
     protected List<String> captureHeaders;
+    protected boolean plainHttpTunneling;
```

Comment:
This config option is specific to local Hoverfly only, it should be moved to `LocalHoverflyConfig`.

Reply:
Ah, right. Moved it there now.